conf/test_coidb.config

```diff
@@ -0,0 +1,34 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for running minimal tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines input files and everything required to run a fast and simple pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/ampliseq -profile test_coidb,<docker/singularity> --outdir <OUTDIR>
+
+----------------------------------------------------------------------------------------
+*/
+
+params {
+    config_profile_name = 'Test COIDB DADA2 reference taxonomy database profile'
+    config_profile_description = 'Minimal test dataset to check --dada_ref_taxonomy with COIDB'
+
+    // Limit resources so that this can run on GitHub Actions
+    max_cpus   = 2
+    max_memory = '6.GB'
+    max_time   = '6.h'
+
+    // Input data
+    FW_primer = "GTGYCAGCMGCCGCGGTAA"
+    RV_primer = "GGACTACNVGGGTWTCTAAT"
+    input = "https://raw.githubusercontent.com/nf-core/test-datasets/ampliseq/samplesheets/Samplesheet.tsv"
+
+    // Custom reference taxonomy
+    dada_ref_taxonomy = "coidb"
+    // Don't add species
+    skip_dada_addspecies = false
```

Comment:
If you want to skip it, that needs to be true ;-)

Reply:
Ah yes, that will be because the file is too big. I'll see if I can get a test version ready.

Reply:
I thought I would test that step, seeing how the addSpecies file is generated by the coidb tool.

Reply:
@d4straub I added a line to the changelog. Is there some docs page that should get updated as part of this PR?

Reply:
> I thought I would test that step, seeing how the addSpecies file is generated by the coidb tool.

I see, but then the comment `// Don't add species` seems misleading? Also, you could just delete those 2 lines completely because `false` is default, if I am not mistaken?

About the docs:
Would be probably nice to add it to the summary figure, e.g. add CO1 in [ampliseq_workflow.svg](https://raw.githubusercontent.com/nf-core/ampliseq/2.4.1/docs/images/ampliseq_workflow.svg) &  [ampliseq_workflow.png](https://raw.githubusercontent.com/nf-core/ampliseq/2.4.1/docs/images/ampliseq_workflow.png) (CO1 would then include CO1DB & Midori2-co1)

Reply:
>I see, but then the comment // Don't add species seems misleading? Also, you could just delete those 2 lines completely because false is default, if I am not mistaken?

Yes very true. Removing those lines.

>Would be probably nice to add it to the summary figure, e.g. add CO1 in [ampliseq_workflow.svg](https://raw.githubusercontent.com/nf-core/ampliseq/2.4.1/docs/images/ampliseq_workflow.svg) & [ampliseq_workflow.png](https://raw.githubusercontent.com/nf-core/ampliseq/2.4.1/docs/images/ampliseq_workflow.png) (CO1 would then include CO1DB & Midori2-co1)

I'm not sure I feel comfortable with updating that overview

Reply:
> I'm not sure I feel comfortable with updating that overview

Alright, then leave it be, I'll do the eventually. But I dont think you can break something ;-)

Reply:
As for the test data. I realize that the typical DADA2 reference taxonomy files contain some 20-40 k sequences whereas the COI references such as COIDB and midori2 have >2M sequences. So creating a 10% test fasta for COIDB was still too big for the testing